src/Publisher/Amazon/SNS/Application/PlatformEndpoint/Publish.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ *  The MIT License (MIT)
```

Comment:
MIT means free software, which is not the case here, please find suitable licence title & text (below)